src/lib.rs

```diff
@@ -69,15 +69,22 @@ pub struct DebuggerLogger {
 /// info!("Hello, world!");
 /// debug!("Hello, world, in detail!");
 /// ```
-pub static DEBUGGER_LOGGER: DebuggerLogger = DebuggerLogger { _priv: () };
+pub static DEBUGGER_LOGGER: DebuggerLogger = DebuggerLogger { force_log_without_debugger: std::sync::atomic::AtomicBool::new(false) };
```

Comment:
I think this exceeds the column max width.  Please run `cargo fmt`.

Reply:
Ok